docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -101,7 +101,7 @@ Installing OpenVINO Runtime from archive files is recommended for C++ developers
 
 Congratulations, you finished the installation! The `/opt/intel/openvino_2022` folder now contains the core components for OpenVINO. If you used a different path in Step 2, for example, `/home/<USER>/Intel/`, OpenVINO is then installed in `/home/<USER>/Intel/openvino_2022`.
 
-The installation directory above is also referred as <INSTALL_DIR> throughout the documentation.
+The directory containing the `openvino_2022` folder is also referred as `<INSTALL_DIR>` throughout the OpenVINO documentation.
```

Comment:
```suggestion
The `openvino_2022` directory is also referred to as `<INSTALL_DIR>` throughout the OpenVINO documentation.
```
The directory containing the `openvino_2022` folder is `/home<USER>/Intel` or `/opt/intel` in the example above, which is not what we refer to with INSTALL_DIR.

Reply:
Yeah, I was going to say `<INSTALL_DIR>` includes the `openvino_2022` folder. Now I see the confusion. Thanks for pointing it out.